src/main/java/apoc/load/util/LoadJdbcConfig.java

```diff
@@ -21,10 +23,23 @@ public LoadJdbcConfig(Map<String,Object> config) {
         } catch (DateTimeException e) {
             throw new IllegalArgumentException(String.format("The timezone field contains an error: %s", e.getMessage()));
         }
+        this.credentials = config.containsKey("credentials") ? validateCredentials((Map<String, Object>) config.get("credentials")) : Collections.emptyMap();
     }
 
     public ZoneId getZoneId(){
         return this.zoneId;
     }
 
+    public Map<String,Object> getCredentials() {
+        return this.credentials;
+    }
+
+    private Map<String, Object> validateCredentials(Map<String,Object> credentials) {
+        if (!credentials.get("user").equals("") && credentials.get("user") != null && !credentials.get("password").equals("") && credentials.get("password") != null) {
```

Comment:
Così com'è potrebbe dare `NullPointerException` in caso non ci sia una chiave, ovvero in caso sia `{user: 'foo'}` o `{user: ''}` o `{password: 'bar'}` o `{password: ''}`

```suggestion
        if (credentials.getOrDefault("user", StringUtils.EMPTY).equals(StringUtils.EMPTY) || credentials.getOrDefault("password", StringUtils.EMPTY).equals(StringUtils.EMPTY)) {
```
Cerchiamo sempre di utilizzare **Costanti** tipo `StringUtils.EMPTY`...